audiocraft/losses/sisnr.py

```diff
@@ -41,6 +41,9 @@ class SISNR(nn.Module):
 
     Input should be [B, C, T], output is scalar.
 
+    ..Warning:: We report here minus SISNR, so negative values are better. This is so that
+        this can be naturally used as a loss!
+
```

Comment:
@adefossez  i kindly request to update the following warning messages, these updates are based on my understanding and aim to avoid confusion when reading the documentation. i apologize for any inconvenience caused by my lack of understanding. thank you for considering my request. 

eg: ..Warning:: The reported values presented here represent the negation of SISNR. Consequently, negative values indicate better results. This format is intentionally designed to facilitate the convenient usage of SISNR as a loss function.

Reply:
i have extended a bit the warnings